test/functional/api/cas/cache.py

```diff
@@ -173,3 +173,6 @@ def get_cache_config(self):
         return CacheConfig(self.get_cache_line_size(),
                            self.get_cache_mode(),
                            self.get_cleaning_policy())
+
+    def standby_detach(self, shortcut: bool = False):
+        casadm.standby_detach_cache(cache_id=self.cache_id, shortcut=shortcut)
```

Comment:
`casadm.standby_detach_cache()` returns an object with result, stdout and stderr. You need to pass it to the caller.

Reply:
Adding return will be enaugh? I don't see value in using it in this specific test.

Reply:
I agree it is unnecessary in this test, but in general API should be consistent